.github/workflows/test.yaml

```diff
@@ -18,3 +18,9 @@ jobs:
           go-version: "1.19"
       - name: Run unit tests
         run: go run mage.go test:go
+      - name: Install goveralls
+        run: go install github.com/mattn/goveralls@latest
+      - name: Send coverage
+        env:
+          COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: goveralls -coverprofile=coverage.txt -service=github
```

Comment:
[nit] please separate the steps from each other. This compact definitions are hard to read